students/20th/seyongyun/js/login.js

```diff
@@ -0,0 +1,31 @@
+const thisIsButton = document.getElementsByClassName("loginButton")[0];
+const inputId = document.getElementsByClassName("idInput")[0];
+const inputPw = document.getElementsByClassName("passwordInput")[0];
+
+inputId.addEventListener("keyup", function() {
+
+    let inputIdLength = inputId.value.length;
+    let inputPwLength = inputPw.value.length;
+
+    if (inputIdLength>1 && inputPwLength>1) {
+        thisIsButton.style.backgroundColor = "blue"
+    }
+
+    if (inputIdLength<2 || inputPwLength<2) {
+        thisIsButton.style.backgroundColor = "rgb(175, 217, 245)"
+    }
+})
+
+inputPw.addEventListener("keyup", function() {
+
+    let inputIdLength = inputId.value.length;
+    let inputPwLength = inputPw.value.length;
+
+    if (inputIdLength>1 && inputPwLength>1) {
+        thisIsButton.style.backgroundColor = "blue"
+    }
+
+    if (inputIdLength<2 || inputPwLength<2) {
+        thisIsButton.style.backgroundColor = "rgb(175, 217, 245)"
+    }
+})
```

Comment:
addEventListener 의 인자로 넘겨지는 함수를 정의해주어, 반복을 줄여보세요!

```suggestion
function enableButton () {
   ...
}

inputId.addEventListener("keyup", enableButton);
```